docs/source/basics.rst

```diff
@@ -369,10 +369,26 @@ The ``CONTROLLED`` modifier takes a gate and makes it a controlled gate. For ins
 .. note::
     The letter ``C`` in the gate name has no semantic significance in Quil. To make a controlled ``Y`` gate, one `cannot` write ``CY``, but rather one has to write ``CONTROLLED Y``.
 
+The ``FORKED`` modifier allows for a parametric gate to be applied, with the specific choice of parameters conditional on a qubit value. For a parametric gate ``G`` with k parameters,
+
+.. parsed-literal::
+
+    FORKED G(u1, ..., uk, v1, ..., vk) c q1 ... qn
+
+is equivalent to
+
+.. parsed-literal::
+
+    if c == 0:
+        G(u1, ..., uk) q1 ... qn
+    else if c == 1:
+        G(v1, ..., vk) q1 ... qn
+
+extended by linearity for general c. Note that the total number of parameters in the forked gate has doubled.
```

Comment:
Nice. I tried to think of a concise way to say "extended by linearity for general c" when I wrote the Quil spec section for FORKED, but failed. Shoulda asked the experts.